internal/cli/cmd/dac/build/testdata/go/go.mod

```diff
@@ -13,9 +13,9 @@
 
 module dac
 
-go 1.22.4
+go 1.22.5
 
-toolchain go1.22.5
+toolchain go1.23.0
```

Comment:
you can revert this change, it's not related to the PR ;)

Reply:
that would solve the current conflict with the `main` branch

Reply:
Yep, IntelliJ doing is stuff